web/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/selectors/CSSSelectors.kt

```diff
@@ -7,19 +7,19 @@ import org.jetbrains.compose.web.css.SelectorsScope
 internal const val webCssSelectorsDeprecationMessage = "Consider using a property from SelectorsScope"
 private val selectorScope = object : SelectorsScope {}
 
-sealed class Nth {
-    data class Functional(val a: Int? = null, val b: Int? = null) {
+sealed interface Nth {
+    data class Functional(val a: Int? = null, val b: Int? = null) : Nth {
```

Comment:
Not for this PR, but might be good to decide before next build:

since previous API was not very useful (didn't work as expected), can we make it internal and provide corresponding public functions in SelectorsScope?

Reply:
I suggest not to do it in this PR since it introduce breaking changes (while this one just fix current API) but I totally agree-  it's just that such selctor API steel need to be designed.